src/css/custom.scss

```diff
@@ -32,4 +32,80 @@
     max-width: 50rem;
     margin: auto;
   }
+
+  .bs-chart,
+.bs-chart__cards,
+.bs-chart__card {
+  display: flex;
+}
+
+.bs-chart {
+  margin: 40px 0;
+  justify-content: space-between;
```

Comment:
I think with 'space-between', this ends up looking a little funky:

![Screenshot 2023-01-26 at 1 03 27 PM](https://user-images.githubusercontent.com/1930213/214914404-798381af-220b-4d57-a99a-71beadeb3198.png)

I think 'space-between' in addition to the content for IE11 being left justified in its container is giving it a bit of an off look.

If we switch this to 'center':
```suggestion
  justify-content: center;
```
![Screenshot 2023-01-26 at 1 06 44 PM](https://user-images.githubusercontent.com/1930213/214914633-2bed871f-eae7-41a5-9236-5d2c04922f5c.png)

That doesn't fix the text though 🤔

Reply:
I already removed `space-between` in favor of not setting it, but I think `center` looks a little nicer so I'll add that back

the text is a little funny, but I'm alright with that for now if you are haha

Reply:
<img width="695" alt="Screenshot 2023-01-26 at 1 30 03 PM" src="https://user-images.githubusercontent.com/6207644/214919785-e4c93dce-feab-4e1c-9d54-3a2f35647176.png">

^^ this is what it looks like with `justify-content: center` -- IE is sort of set off to the side haha